candle-kernels/src/unary.cu

```diff
@@ -1,91 +1,104 @@
 #define _USE_MATH_DEFINES
-#include<math.h>
-#include<stdint.h>
+#include <math.h>
```

Comment:
I didn't realize my editor auto formatted this, lmk if this is undesireable and I'll undo

Reply:
Yeah might be worth removing all the formatting changes if it's easy on your side, this will make the diff easier to look at.